Sources/General/ZLTextStickerView.swift

```diff
@@ -286,6 +288,21 @@ class ZLTextStickerView: UIView, ZLStickerViewAdditional {
             delegate?.stickerEndOperation(self, panGes: panGes)
         }
     }
+
+    func updateFontSize() {
```

Comment:
Thanks for your pr.
This method has some UI experience is not very good, and it does not perfectly adapt to various situations.
So I added a maximum scale to the stickers to avoid blurry fonts.

Reply:
Personally I don't think this is bad, as you can achieve interesting effects where the label is bigger than the screen and fonts are cut off